### ISSUE TITLE
create another language

### ISSUE THREAD

**Issue by Karaulov** [actboy168/YDWE#118]
which text files are responsible for which languages, what needs to be changed to add a new language?

**Comment by Karaulov** [comment]
![82cnoe8 1](https://user-images.githubusercontent.com/6445336/34640107-a5a0283a-f2fd-11e7-9665-fdeb6f315443.png)

....

**Comment by actboy168** [comment]
You can found the ui file from 'ydwe/share/mpq/'.

**Comment by Karaulov** [comment]
YDWE support only single language ? (can't create multiple languages and select it in settings)


I try to replace one of files with another language, but got error messages 'ydwe*****' string not found, etc

**Comment by actboy168** [comment]
Yes.

**Comment by Karaulov** [comment]
I try to replace one of files with another language, but got error messages 'ydwe*****' string not found, etc

How to find manual for edit language files? How to edit language in YDWE config menu?

**Comment by Unryze** [comment]
I am interested in this as well and I am ready to translate YDWE to English and Russian (can do even Spanish if needed, but preferably just those 2 languages). All that is really needed is to add the possibility of choosing language.

I hope for a positive answer, thanks!

**Comment by Karaulov** [comment]
Yes, @actboy168 , you can add multiple ydwe/share/mpq/ directories for different languages and config for select language ?

**Comment by actboy168** [comment]
OK.I will add support for multiple languages.

**Comment by Karaulov** [comment]
This only for YWDE menu ?

**Comment by actboy168** [comment]
@Karaulov  Many things are in process.

**Comment by Unryze** [comment]
@actboy168 Thank you very much for understanding and thank you for doing what you do!

**Comment by actboy168** [comment]
YDWE already supports multiple languages. All configurations are in ydwe/share/.I added some simple English translation, if you need to modify or add a new language, you can submit the pull request.

@Unryze @Karaulov

**Comment by Karaulov** [comment]
Config translate not works ?


Config with CN lang

![eri2ajw 1](https://user-images.githubusercontent.com/6445336/34865090-627fec90-f788-11e7-9450-555d298abf22.png)

Config with EN lang

![jmk23ct 1](https://user-images.githubusercontent.com/6445336/34865098-6e174ec2-f788-11e7-8b37-c74a1a023d2d.png)

**Comment by Karaulov** [comment]
And with EN language no default World Editor Triggers translate, how to create it (or this is error) ?

![7gfw6vw 1](https://user-images.githubusercontent.com/6445336/34865174-b5afb74c-f788-11e7-9159-0efb73ea64f8.png)

**Comment by actboy168** [comment]
This is because there is no translation yet.Config translate is in share/skins/en-US and UI translate is in share/ui/xxx/en-US.

**Comment by Karaulov** [comment]
I don't know english, try create test translate using https://www.bing.com/translator and google translate :)



Some text are cutted.  You can increase text length ?

![egzd5x8 1](https://user-images.githubusercontent.com/6445336/34866725-97328320-f78e-11e7-9797-9762ea754b06.png)

**Comment by actboy168** [comment]
```
<RadioButton class="NormalRadioButton" text="只保存为旧版本（1.20）" width="300" group="MapSave" name="MapSave_1" tooltip="保存为适用于魔兽1.24补丁以前的版本。" />
<RadioButton class="NormalRadioButton" text="只保存为新版本（1.24）" width="200" group="MapSave" name="MapSave_2" tooltip="保存为适用于魔兽1.24补丁以及以后的版本。" />
```
You can modify the 'width' field.

**Comment by Unryze** [comment]
I have translated menu and inserted WorldEditor Strings from English Warcraft 3, all seems to be fine. 

There is one issues though, 
![screenshot_1](https://user-images.githubusercontent.com/11413837/42229519-b75091b0-7ef7-11e8-9c50-516349a4918b.png)
![screenshot_2](https://user-images.githubusercontent.com/11413837/42229521-b772d112-7ef7-11e8-90c5-2cde444a2d34.png)
[ENDialogConfig.zip](https://github.com/actboy168/YDWE/files/2160027/ENDialogConfig.zip)

I am too lazy to translate GUI, as I never use it, so I did not touch it yet, anyways, as for the rest, it works as it should ;)
